testsuite/tests/basic-more/opaque_prim.ml

```diff
@@ -0,0 +1,8 @@
+
+let f x = opaque x
+
+let () =
+  assert(f f == f);
+  assert(opaque 1 = 1);
+  assert(opaque 1. = 1.)
+
```

Comment:
No, it's not really possible. It mainly test that this primitive is handled by the code generator since it is quite easy to add a primitive in lambda and forget it elsewhere.